hack/gen_rvar_table.py

```diff
@@ -0,0 +1,119 @@
+#!/bin/python
+
+import re
+
+with open('defaults/main.yml') as f:
+    lines = f.read().split("\n")
+
+def print_rvar_table(rvar_table):
+    keys = ["name", "description", "default", "required"]
+
+    sizes = {}
+    for key in keys:
+        sizes[key] = len(key)
+
+    for item in rvar_table:
+        for key in item.keys():
+            sizes[key] = max(sizes[key], len(item[key]))
+
+    headers = []
+    delimiters = []
+    for key in keys:
+        headers.append(" {}{} ".format(key, " "*(sizes[key]-len(key))))
+        delimiters.append("-{}-".format("-"*(sizes[key])))
+    print "|{}|".format("|".join(headers))
```

Comment:
`from __future__ import print_function` at the top would be required for 2/3 compat, correct?